packages/bot-skeleton/src/scratch/hooks/data_category.js

```diff
@@ -70,27 +70,44 @@ Blockly.DataCategory.search = function (variableModelList) {
 };
 
 /**
- * Construct a create variable button and push it to the xmlList.
+ * Construct a create variable section and push it to the xmlList.
  * @param {!Array.<!Element>} xmlList Array of XML block elements.
  * @param {Blockly.Workspace} workspace Workspace to register callback to.
  * @deriv/bot: We only use a single type of variable, so `type` arg was removed.
  */
-Blockly.DataCategory.addCreateButton = function (xmlList, workspace) {
+Blockly.DataCategory.addCreateVariable = function (xmlList, workspace) {
     const buttonXml = goog.dom.createDom('button');
+    const inputXml = goog.dom.createDom('input');
+
     // Set default msg, callbackKey, and callback values for type 'VARIABLE'
-    const msg = localize('Create variable');
+    const btnMsg = localize('Create');
```

Comment:
Can we use snake case and expand this so it's more clear?

```suggestion
    const button_message = localize('Create');
```